app/video_routes.py

```diff
@@ -0,0 +1,121 @@
+from app import db
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+from datetime import datetime
+from flask import Blueprint, jsonify, request
+
+
+videos_bp = Blueprint("videos", __name__, url_prefix=("/videos"))
+
+
+@videos_bp.route("", methods=["GET"])
+def get_all_videos():
+    if request.method == "GET":
+        video = Video.query.all()
+        video_response = [video.video_information() for video in video]
+    return jsonify(video_response), 200
+
+
+@videos_bp.route("", methods=["POST"])
+def create_video():
+    request_body = request.get_json()
+    if "title" not in request_body:
+        return jsonify(details="Request body must include title."), 400
+
+    if "release_date" not in request_body:
+        return jsonify(details="Request body must include release_date."), 400
+
+    if "total_inventory" not in request_body:
+        return jsonify(details="Request body must include total_inventory."), 400
```

Comment:
you could create a function to handle these guard clauses dynamically and then use that function for customers and videos it could look something like
```
def validate_parameters(request_body, parameters):
           for attribute in request_body:
                if attribute not in request_body:
                       abort(make_response({"details": f"Request body must include {attribute}."}, 400))
```